src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/config/AuditConfig.java

```diff
@@ -383,4 +383,16 @@ public String toString() {
                 .filter(settings::hasValue)
                 .collect(Collectors.toSet());
     }
+
+    public static final ImmutableSet<String> FIELD_PATHS = ImmutableSet.of(
+            "/enable",
+            "/audit/enable_rest", "/audit/disabled_rest_categories",
+            "/audit/enable_transport", "/audit/disabled_transport_categories",
+            "/audit/resolve_bulk_requests", "/audit/log_request_body", "/audit/resolve_indices", "/audit/exclude_sensitive_headers",
+            "/audit/ignore_users", "/audit/ignore_requests",
+            "/compliance/enabled",
+            "/compliance/external_config", "/compliance/internal_config",
+            "/compliance/read_metadata_only", "/compliance/read_watched_fields", "/compliance/read_ignore_users",
+            "/compliance/write_metadata_only", "/compliance/write_log_diffs", "/compliance/write_watched_indices", "/compliance/write_ignore_users"
+    );
```

Comment:
Can we generate this list dynamically instead of hard-coding it?  
This is not flexible long-term in case if we add new configs.

Reply:
Done. Using Jackson to find all the properties of a given class.